examples/Multi_Armed_Bandit/bernoulli_multi_armed_bandit.py

```diff
@@ -0,0 +1,175 @@
+
+
+### Setup
+
+# For our example, we use a system with a fixed collection of win rates, which we
+# assume are unknown to us:
+
+WIN_RATES = [0.3, 0.4, 0.6, 0.65, 0.8]
+BANDIT = MultiArmedBandit(WIN_RATES)
+
+# Given this target system, we consider the following two tasks:
+# - **Online optimization:** Maximizing the total reward accumulated during a campaign
+# - **Active learning:** Estimating the win rates of all arms
+
+# Solving these task can be achieved through an appropriate choice of acquisition function:
+# For online optimization, we use {class}`~baybe.acquisition.acqfs.qThompsonSampling`.
+# For active learning, we switch to {class}`~baybe.acquisition.acqfs.PosteriorStandardDeviation`.
+
+# To estimate the corresponding effects, we simulate each campaign for a certain number of steps and repeat this process in multiple Monte Carlo runs:
+
+ACQFS = [
+    qThompsonSampling(),  # Online optimization
+    PosteriorStandardDeviation(),  # Active learning
+]
+N_MC_RUNS = 10
+N_ITERATIONS = 200
+
+
+### Building the Model
+
+# To model the rewards obtained from the bandit, we use a {class}`~baybe.targets.binary.BinaryTarget`.
+# The corresponding search space is spanned by a {class}`~baybe.parameters.categorical.CategoricalParameter` describing the bandit's arms.
+
+n_arms = len(WIN_RATES)
+objective = BinaryTarget(name="clicked").to_objective()
+searchspace = CategoricalParameter(
+    name="Bandit Arm", values=[f"arm_{i}" for i in range(n_arms)]
+).to_searchspace()
+surrogate = BernoulliMultiArmedBanditSurrogate()
+
+
+# For each simulation, we report the trajectory of earned rewards and the estimated win rates of the bandit arms:
+
+
+class SimulationResult(NamedTuple):
+    """Result of a bandit campaign simulation."""
+
+    earned_rewards: np.ndarray
+    estimated_win_rates: np.ndarray
+
+
+# Putting all pieces together, we define a function to simulate our campaign under a given acquisition strategy:
+
+
+def simulate(acqf: AcquisitionFunction) -> SimulationResult:
+    """Simulate the campaign with with the given acquisition function."""
+    recommender = TwoPhaseMetaRecommender(
+        initial_recommender=RandomRecommender(
+            allow_repeated_recommendations=True,
+            allow_recommending_already_measured=True,
+        ),
+        recommender=BotorchRecommender(
+            surrogate_model=surrogate,
+            acquisition_function=acqf,
+            allow_repeated_recommendations=True,
+            allow_recommending_already_measured=True,
+        ),
+    )
+
+    # Containers for storing the rewards and win rate estimates
+    earned_rewards = np.zeros((N_MC_RUNS, N_ITERATIONS))
+    estimated_win_rates = np.zeros((N_MC_RUNS, n_arms))
+
+    for mc in range(N_MC_RUNS):
+        campaign = Campaign(searchspace, objective, recommender)
+
+        for i in range(N_ITERATIONS):
+            df = campaign.recommend(batch_size=1)
+            recommended_arm_idx = int(df.iloc[0, 0].split("_")[1])
```

Comment:
If we'd give simpler names to the arms (like just numbering them) this part of the code would be significantly easier to understand.

Reply:
Yes, also thought about that because I didn't like this consequence. I guess I'll change

Reply:
done